packages/jest-preset-default/scripts/travis-fold-passes-reporter.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+const { DefaultReporter } = require( '@jest/reporters' );
+
+class TravisFoldPassesReporter extends DefaultReporter {
+	constructor( ...args ) {
+		super( ...args );
+		this.foldedTestResults = [];
+	}
+
+	flushFoldedTestResults() {
+		if ( this.foldedTestResults.length > 0 ) {
```

Comment:
Could be an early return to unindent the function:

```js
if ( ! this.foldedTestResults.length ) {
	return;
}
```

Reply:
Good call.